cli/pkg/kctrl/cmd/package/release/imgpkg_runner.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2022 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package release
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	goexec "os/exec"
+	"path/filepath"
+	"strings"
+)
+
+type ImgpkgRunner struct {
+	Image             string
+	Version           string
+	Paths             []string
+	UseKbldImagesLock bool
+	ImgLockFilepath   string
+}
+
+func (r ImgpkgRunner) Run() (string, error) {
+	dir, err := os.MkdirTemp(".", fmt.Sprintf("bundle-%s-*", strings.Replace(r.Image, "/", "-", 1)))
+	if err != nil {
+		return "", err
+	}
+
+	wd, err := os.Getwd()
+	for _, path := range r.Paths {
+		var stderrBuf bytes.Buffer
+		cmd := goexec.Command("cp", "-r", filepath.Join(wd, path), dir)
+		cmd.Stderr = &stderrBuf
+		err := cmd.Run()
+		if err != nil {
+			return "", fmt.Errorf("%s", stderrBuf.String())
+		}
+	}
```

Comment:
copying should not have to use an external command e.g. https://github.com/vmware-tanzu/carvel-vendir/blob/develop/pkg/vendir/directory/directory.go#L168-L171

Reply:
Using the lib now!